File: mario.py
import cs50

while True:
    height = cs50.get_int()
    print("Height: {}".format(height))
    if height >= 0 or height <=23:
        break

for i in range(height):
    j=0
    for j in range(height - i - 1):
        print(" ", end="")
    print("#" * i, end="")
    print("  ", end="")
    print("#" * i)
        
        

File: mario.c
#include <stdio.h>
#include <cs50.h>


/*

Prompt user for a number. Print a pyramid of that height. 
Ex: Input: 2, Output:  # #
                      ## ##
*/


int main(void)
{
    int height;
    do  
    {
    printf("Height: ");
    height = get_int();
    }
    while (height < 0 || height > 23);
    
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
            }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
            }
        printf("  ");
        for (int j = 0; j <= i; j++)
        {
            printf("#");
            }
        printf("\n");
    }
}

